demisto_sdk/commands/prepare_content/tests/list_unifier_test.py

```diff
@@ -0,0 +1,120 @@
+from pathlib import Path
+
+import pytest
+
+from demisto_sdk.commands.common.constants import LISTS_DIR
+from demisto_sdk.commands.common.tools import pascal_case
+from demisto_sdk.commands.prepare_content.list_unifier import ListUnifier, logger
+from demisto_sdk.tests.test_files.validate_integration_test_valid_types import (
+    LIST,
+)
+from TestSuite.test_tools import ChangeCWD
+
+
+def create_split_list():
+    list_json = LIST
+    list_data = LIST["data"]
+    list_json["data"] = "-"
+    list_dir_name = Path(LISTS_DIR) / pascal_case(list_json["name"])
+    return list_json, list_data, list_dir_name
+
+
+def test_list_unify(git_repo):
+    '''
+    Given:
+        - A list json file.
+    When:
+        - Running ListUnifier.unify on it.
+    Then:
+        - Ensure the list is unified as expected.
+    '''
+    pack = git_repo.create_pack("PackName")
+    list_json, list_data, list_dir_name = create_split_list()
+    list_dir_path = Path(pack.path) / list_dir_name
+    list_dir_path.mkdir(parents=True, exist_ok=True)
+    (list_dir_path / (list_dir_name.name + ".json")).write_text(str(list_json))
+    (list_dir_path / (list_dir_name.name + "_data.css")).write_text(str(list_data))
+
+    with ChangeCWD(pack.repo_path):
+        list_unifier = ListUnifier.unify(
+            path=list_dir_path / (list_dir_name.name + ".json"), data=list_json
+        )
+        assert list_unifier["data"] == list_data
+
+
+@pytest.mark.parametrize("suffix", ["css", "json", "txt", "md", "csv", "html"])
+def test_find_file_content_data(tmpdir, suffix):
+    '''
+    Given:
+        - A data file with a suffix that is txt, json, css, md, csv or html.
+    When:
+        - Running `find_file_content_data` method.
+    Then:
+        - Ensure the path of the data file is returned.
+    '''
+    (Path(tmpdir) / f"_data.{suffix}").write_text("data")
+    assert (
+        ListUnifier.find_file_content_data(Path(tmpdir) / "_data")
+        == Path(tmpdir) / f"_data.{suffix}"
+    )
```

Comment:
```suggestion
    path = Path(tmpdir, f"_data.{suffix}")
    path.write_text("data")
    assert (
        ListUnifier.find_file_content_data(path) == path
    )
```

reuse, reduce, recycle